packages/auth/src/Auth.ts

```diff
@@ -304,18 +305,36 @@ export default class AuthClass {
 
     /**
      * Sign in
-     * @param {String} username - The username to be signed in
+     * @param {String | Object} username - The username to be signed in

```

Comment:
I think Typescript understands types in JSDoc blocks, see [JSDoc-support-in-JavaScrip](https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript#param-and-returns)

Also, adjust docs to note that the first param can be a username or `SignInOpts`

```suggestion
     * @param {String | SignInOpts} username - The username to be signed in
```

Reply:
Done.